src/authorizeStep.js

```diff
@@ -18,8 +18,8 @@ export class AuthorizeStep {
         return next.cancel(new Redirect(loginRoute));
       }
     }
-    else if (isLoggedIn && routingContext.getAllInstructions().some(i => i.fragment) == loginRoute) {
-      var loginRedirect = this.auth.getLoginRedirect();
+    else if (isLoggedIn && routingContext.getAllInstructions().some(i => i.fragment == loginRoute)) {
+      var loginRedirect = this.auth.getLoginRedirect().replace(/\/#/gi, '');
```

Comment:
You're right. I just didn't want to make a breaking change. I'll make a better fix and make a different pull request. Thanks!